.travis.yml

```diff
@@ -1,16 +1,18 @@
 language: node_js
 node_js:
 - 6.6.0
+sudo: false
 before_install:
-- npm install -g mocha
-- npm install -g gulp
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
https://yarnpkg.com/en/docs/install-ci#travis-tab, prior to adding this before_install line - the build failed on `yarn global add mocha`.